{"name": "test_login_1_failed_error_data[data1]", "status": "passed", "attachments": [{"name": "log", "source": "d05845df-18b9-4811-978b-01ad93a52098-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "7ab897c7-261d-4e2e-8750-9c19203523f1-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "data", "value": "{'user': '18684720553', 'passwd': 'python11', 'check': '帐号或密码错误!'}"}], "start": 1608132625942, "stop": 1608132626716, "uuid": "aef55bb7-8df3-477c-a2d1-503341eb458a", "historyId": "308aaeb32698655c72872f8225ec1a01", "testCaseId": "1fc2970947168ce03ea67fcd12d25389", "fullName": "testcase.test_1login_pytest.TestLogin#test_login_1_failed_error_data", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('refresh_page')"}, {"name": "tag", "value": "login"}, {"name": "parentSuite", "value": "testcase"}, {"name": "suite", "value": "test_1login_pytest"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-J6CF24D"}, {"name": "thread", "value": "53180-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcase.test_1login_pytest"}]}